l10n_it_withholding_tax/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 
 {
     'name': 'Italian Withholding Tax',
-    'version': '12.0.1.1.0',
+    'version': '12.0.1.2.0',
```

Comment:
Non serve modificare la versione perchè ci pensa OCA Bot in fase di merge

Reply:
Sistemo subito

Reply:
@OpenCode fatto

Reply:
@OpenCode non è necessario chiedere ai contributors l'aumento di versione perché può essere fatto dal maintainer che mergia la PR con i comandi di ocabot, vedi https://github.com/OCA/oca-github-bot#commands

Reply:
@SimoRubi Non ho chiesto di aumentare la versione. Ho espressamente detto il contrario, cioè di rimuovere la modifica perché ci pensa ocabot e @SilvioGregorini ha fatto proprio quello.

Reply:
ah scusa ho letto male, ritiro tutto :D

Reply:
@SimoRubi Ora per farti perdonare fai review e approve così arriviamo a 4 spunte che fanno bene all'anima!

Reply:
> @SimoRubi Ora per farti perdonare fai review e approve così arriviamo a 4 spunte che fanno bene all'anima!

**Ora** è una parola grossa, comunque è giusto, farò review :)

Reply:
@OpenCode avresti potuto lasciare la versione modificata e mergiare semplicemente con `/ocabot merge`.

Reply:
@tafaRU ceto, ma tanto vale abituarsi a non fare questa modifica se usiamo ocabot perché così evitiamo che PR parallele applichino la stessa modifica